app/backend/templates/donation_received.md

```diff
@@ -0,0 +1,25 @@
+---
+subject: "Thank you for your donation to Couchers.org!"
+---
+
+{% from "macros.html" import button, link, support_email %}
+
+Hi {{ user.name|couchers_escape }},
+
+Thank you for your donation of $ {{ amount }} to the Couchers.org Foundation. Your donation makes our work and the wellbeing of the community possible, and the project wouldn't exist without the support of you and other financial contributors!
```

Comment:
IMO "without your support!"

No need to talk about other people

Reply:
I'm going to update the email with Itsi's proper one